.github/workflows/checks.yml

```diff
@@ -81,3 +81,42 @@ jobs:
         run: npm run typecheck
       - name: Run tests
         run: npm run test:unit
+
+  E2E:
+    # Only run e2e tests, if other jobs are successful
```

Comment:
I'd prefer to run these in parallel with Lint/Test to save time waiting for PR's to finish testing. Also then you can fix all test errors in one commit rather than: fix unit test, commit, see broken e2e test, fix, commit

Reply:
Good point. Should I leave `needs: [Build]` thou? If the build fails, e2e job will certainly fail too.

Reply:
Yea it'd need build to be able to work.